extra-x11/nvidia-prime/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=nvidia-prime
+PKGSEC=x11
+PKGDES="NVIDIA Prime Render Offload configuration and utilities"
+PKGDEP="libglvnd mesa vulkan"
```

Comment:
Is `nvidia-prime` a "binary" or a script? If it's a script, append `ABHOST=noarch` as below (same goes for all packages marked as "any" in Arch Linux.

Reply:
It is a script. Appended `ABHOST=noarch`.